postgres_scanner.cpp

```diff
@@ -715,6 +717,22 @@ static void ProcessValue(data_ptr_t value_ptr, idx_t value_len, const PostgresBi
 		FlatVector::GetData<interval_t>(out_vec)[output_offset] = res;
 		break;
 	}
+
+	case LogicalTypeId::UUID: {
+		D_ASSERT(bind_data->columns[col_idx].attlen == 2 * sizeof(int64_t));
+		D_ASSERT(value_len == 2 * sizeof(int64_t));
+		D_ASSERT(bind_data->columns[col_idx].atttypmod == -1);
+
+		hugeint_t res;
+
+		auto upper = ntohll(Load<uint64_t>(value_ptr));
+		res.upper = upper ^ (int64_t(1) << 63);
```

Comment:
I'm not sure why this is needed. I copied this from https://github.com/duckdb/duckdb/blob/master/src/common/types/uuid.cpp#L52